WordPress/build.gradle

```diff
@@ -169,6 +171,7 @@ android {
             buildConfigField "String", "TRACKS_EVENT_PREFIX", '"wpandroid_"'
             buildConfigField "String", "PUSH_NOTIFICATIONS_APP_KEY", '"org.wordpress.android"'
             buildConfigField "boolean", "ENABLE_QRCODE_AUTH_FLOW", "false"
+            buildConfigField "boolean", "ENABLE_OPEN_WEB_LINKS_WITH_JP_FLOW", "false"
```

Comment:
Does this mean it's disabled for WP and enabled for JP? If so I thought we needed the reverse maybe, wdyt?

Reply:
@develric This has been changed in a subsequent PR and will have no effect until the future.